README.md

```diff
@@ -1030,41 +1034,105 @@ You can replace ad images with pictures of kittens, for example, by starting the
 
 * One or more profile(s) for secure and trusted browsing needs, such as banking and email only.
 
-The idea is to separate and compartmentalize data, so that an exploit or privacy violation in one "session" does not necessarily affect data in another.
+The idea is to separate and compartmentalize data so that an exploit or privacy violation in one "session" does not necessarily affect data in another.
 
 In each profile, visit `chrome://plugins/` and disable **Adobe Flash Player**. If you must use Flash, visit `chrome://settings/contents` to enable **Let me choose when to run plugin content**, under the Plugins section (also known as *click-to-play*).
 
+[Incognito](https://support.google.com/chrome/answer/7440301) mode in Chrome disables extensions, since extensions such as Ad blockers have access to Chrome's network requests. Extensions have to be enabled manually. Moreover, while in Incognito mode, Chrome does not use session data from previous sessions. Incognito mode is another option if you want to access sensitive information without setting up separate profiles.
+
 Take some time to read through [Chromium Security](https://www.chromium.org/Home/chromium-security) and [Chromium Privacy](https://www.chromium.org/Home/chromium-privacy).
 
-For example you may wish to disable [DNS prefetching](https://www.chromium.org/developers/design-documents/dns-prefetching) (see also [DNS Prefetching and Its Privacy Implications](https://www.usenix.org/legacy/event/leet10/tech/full_papers/Krishnan.pdf) (pdf)).
+For example, you may wish to disable [DNS prefetching](https://www.chromium.org/developers/design-documents/dns-prefetching) (see also [DNS Prefetching and Its Privacy Implications](https://www.usenix.org/legacy/event/leet10/tech/full_papers/Krishnan.pdf) (pdf)).
 
-Also be aware of [WebRTC](https://en.wikipedia.org/wiki/WebRTC#Concerns), which may reveal your local or public (if connected to VPN) IP address(es). This can be disabled with extensions such as [uBlock Origin](https://github.com/gorhill/uBlock/wiki/Prevent-WebRTC-from-leaking-local-IP-address) and [rentamob/WebRTC-Leak-Prevent](https://github.com/rentamob/WebRTC-Leak-Prevent).
+It is best to remember that Google is an advertising company and its major source of revenue is [AdSense](https://www.google.com/adsense/start/#/?modal_active=none). It makes perfect sense that an advertising company would leverage its services to maximize its profit. That means that while using [Google services](https://www.google.com/services/#?modal_active=none) certain personal information are being stored. Google is quite open about the data it stores and how it used them. Users can opt out from many of those services and see what type of information Google has stored from their [account settings](https://myaccount.google.com/privacy?pli=1).
```

Comment:
"perfect sense" and "quite open" may not be the best way to put this. It makes the document look like an advertisement for using Google, where it is intended to be a neutral statement of facts (where even the best security facts are opinions in the end, but the neutrality should be front and center).

Replace the references to the Google sales pages, https://www.google.com/adsense/start/#/?modal_active=none and https://www.google.com/services/#?modal_active=none (has the page title "Google Business Solutions") and replace them with something more informative and not focused on Google's customers, rather being useful for Chrome users.

Changing to just "open" with a reference to where they state in a single document the list of data they store on Chrome users (in particular) would be useful.